doc/swish/erlang.tex

```diff
@@ -2979,6 +2979,29 @@ \subsection{String Utilities}
 \var{separator}. When \var{last-separator} is specified, it is used as
 the last separator.
 
+\pagebreak
+% ----------------------------------------------------------------------------
+\defineentry{natural-string<?}
+\defineentry{natural-string-ci<?}
+\begin{procedure}
+  \code{(natural-string<? \var{s$_1$} \var{s$_2$})}\\
+  \code{(natural-string-ci<? \var{s$_1$} \var{s$_2$})}\strut
+\end{procedure}
+\returns{} a boolean
+
+These procedures compare strings \var{s$_1$} and \var{s$_2$} in a way that is
+more natural to humans than the corresponding
+\code{string<?} and \code{string-ci<?} predicates.
+The natural comparison predicates:
+\begin{enumerate}
+ \item ignore leading and trailing whitespace
+ \item treat runs of multiple whitespace characters as a single space
+ \item compare integers numerically, treating \code{-} as a dash
+   if it follows an alphanumeric character or else as negation
```

Comment:
consider using a comma at the end of items 1 & 2, and ", and" at the end of item 3.

Reply:
Much better!